java/client/src/main/java/glide/api/models/commands/LPosOptions.java

```diff
@@ -0,0 +1,45 @@
+package glide.api.models.commands;
+
+import glide.api.commands.SortedSetBaseCommands;
+import lombok.Builder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Optional arguments to {@link ListBaseCommands#lpos()},
+ *
+ * @see <a href="https://redis.io/commands/lpos/">redis.io</a>
+ *//**
+// * Optional arguments to {@link ListBaseCommands#lpos(String, Map, LPosOptions)},
+// * {@link ListBaseCommands#lpos(String, LPosOptions)} and {@link
+// * ListBaseCommands#zaddIncr(String, LPosOptions)}
+ *
+ * @see <a href="https://redis.io/commands/lpos/">redis.io</a>
+ */
+@Builder
+public final class LPosOptions {
+
+    private Long rank;
+    private Long maxLength;
+    private final String COUNT_REDIS_API = "COUNT";
+
+    /**
+     * Converts LPosOptions into a String[].
+     *
+     * @return String[]
+     */
+    public String[] toArgs() {
+        List<String> optionArgs = new ArrayList<>();
+        if (rank != null) {
+            optionArgs.add("RANK" + Long.toString(rank));
```

Comment:
Don't use string concatenation. 
This will create the String: "RANK0".  Even if you put a space, you will have "RANK 0".  
Add two items to the array like https://github.com/aws/glide-for-redis/pull/1524/files#diff-6118fb6c768818f92c96d93eeee2f53e0d8e21a5929a030203034b1da383460aR45

Note: using string concatenation led to the recent bug with set options in Node ;)